### PR TITLE
[DAT-919] remove unnecesary code

### PR DESCRIPTION
**Pull request by manu3210**
## Sumary

- Removed all sample code
- Changed all example names
- Updated Readme with a description about the service

### CODE REVIEW

File: Doppler.MercadoPagoApi.Test/UnitTest1.cs [hunk 1]
```diff
@@ -1,14 +1,7 @@
-using System;
```
Comment (jhoffman-ms) [inline Doppler.MercadoPagoApi.Test/UnitTest1.cs:1]: The same for this

File: Doppler.MercadoPagoApi.Test/IntegrationTest1.cs [hunk 2]
```diff
@@ -1,13 +1,7 @@
-using System;
```
[the remark this answers is not in the document]
Comment (andresmoschini) [reply Doppler.MercadoPagoApi.Test/IntegrationTest1.cs:1]: I think that deleting it is consistent with what are you doing. 

Probably, when you need to create a new test, the filename will be different and more descriptive.

File: demo.http [hunk 3]
```diff
@@ -1,34 +1,9 @@
 # Use [REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) to run these requests
 
 @base=http://localhost:5000
-# @base=https://apis.fromdoppler.com/hello
-# @base=https://apisint.fromdoppler.net/hello
-# @base=https://apisqa.fromdoppler.net/hello
```
[the remark this answers is not in the document]
Comment (manu3210) [reply demo.http:6]: Thanks Andres, now I add that lines with the new URL.

**Comment by jhoffman-ms** [comment]
👍

File: README.md [hunk 4]
```diff
@@ -1,133 +1,3 @@
...
-- **cd-helper-test** `https://apisqa.fromdoppler.net/cd-helper/hooks/{{REEMPLACE-THE-SECRET-HERE}}/`
-
-![dockerhub-webhooks](./docs/dockerhub-webhooks.png)
-
-**IMPORTANT:** You can see the secrets in other Docker Hub repositories, for example in [_CD-Helper_'s one](https://hub.docker.com/repository/docker/dopplerdock/doppler-cd-helper/webhooks).
-
-### Add the stack to Doppler Swarm repository
-
-We should add the stack to [Doppler Swarm repository](https://github.com/MakingSense/doppler-swarm), it is possible using `hello-stack` as reference. Using _search in files_ is recommended to find all the places to update.
+an api to manage payments made with MercadoPago
```
Comment (afantini) [inline README.md:3]:
```suggestion
API for processing payments with Mercado Pago.
```

**Review by afantini — COMMENTED** [review]
I just wanted to suggest a change in [README.md](https://github.com/FromDoppler/doppler-mercadopago-api/blob/main/README.md) and let you know that [`.vscode\launch.json`](https://github.com/FromDoppler/doppler-mercadopago-api/blob/main/.vscode/launch.json) and [`.vscode\tasks.json`](https://github.com/FromDoppler/doppler-mercadopago-api/blob/main/.vscode/tasks.json) still have references to `HelloMicroservice`. Also, I think that [`docs`](https://github.com/FromDoppler/doppler-mercadopago-api/blob/main/docs/) is no longer necessary.

**Pull request merged**